lib/renderer.js

```diff
@@ -272,6 +272,10 @@
      * Get SVG data for the given component.
      * 
      * @private
+     * @param {Component} component with component.layer or component.comp, or neither.
+     *      If neither is specified, this method gets the SVG data for the whole document.
+     * @param {?Layer} component.layer Layer to get the SVG data for. Exclusive of component.comp.
```

Comment:
Nice. I'll add something like that based on the closure syntax and remove some English :)